packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3997,7 +3997,10 @@ export default class Meeting extends StatelessWebexPlugin {
           canAdmitParticipant: MeetingUtil.canAdmitParticipant(this.userDisplayHints),
           canLock: MeetingUtil.canUserLock(this.userDisplayHints),
           canUnlock: MeetingUtil.canUserUnlock(this.userDisplayHints),
-          canShareWhiteBoard: MeetingUtil.canShareWhiteBoard(this.userDisplayHints),
+          canShareWhiteBoard: MeetingUtil.canShareWhiteBoard(
```

Comment:
we should add a test for this in packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

describe('#updateMeetingActions'